### DISCUSSION TITLE
CYBERDUCK SHowing ZERO FILES

### DISCUSSION THREAD

**Discussion by GGionata** [iterate-ch/cyberduck#15322] [Q&A]
Hello I have just purchased the lates version of CYBERDUCK.
I have installed successfully and downloaded a configuration file provided by my WEB HOSTING platform, REGISTER

It does connect to my FTP and website but in the directory there's nothing 

my website is

www.studio360showroom.com

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), connect to the server, wait until the file listing process is finished, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15322 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.
